sim/hunter/runes.go

```diff
@@ -382,16 +383,23 @@ func (hunter *Hunter) applyHitAndRun() {
 		hunter.HitAndRunAura = hunter.RegisterAura(core.Aura{
 			Label:    "Hit And Run",
 			ActionID: core.ActionID{SpellID: 440533},
-			Duration: time.Second * 8,
+			Duration: time.Second * 15,
 			OnReset: func(aura *core.Aura, sim *core.Simulation) {
 				aura.Activate(sim)
 			},
 			OnGain: func(aura *core.Aura, sim *core.Simulation) {
-				hunter.Unit.MoveSpeed *= 1.15
+				hunter.Unit.MoveSpeed *= 1.30
 			},
 			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
-				hunter.Unit.MoveSpeed *= 1 / 1.15
+				hunter.Unit.MoveSpeed *= 1 / 1.30
 			},
 		})
 	}
 }
+
+func (hunter *Hunter) applyImprovedVolley() {
+	if hunter.HasRune(proto.HunterRune_RuneCloakImprovedVolley) {
+		// The 3% rAP scaling is applied inside the volley spell config itself
+		hunter.Volley.DamageMultiplier *= 2
```

Comment:
I'm not sure whether or not Volley will actually be initialized at this point. Usually i have to make these changes inside an aura